Proyecto2.5/Assets/scripts/Disparo.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using UnityEngine;
+public class Disparo : MonoBehaviour
+{
+    [SerializeField] private GameObject bala;
+    [SerializeField] private Transform direccion;
+    [SerializeField] private float cantidad_de_balas = 10f;
```

Comment:
Format all variables like `_amountOfBullets`, in all scripts, pay attention to the rules.